plugin/plaintextmodel.cpp

```diff
@@ -264,6 +277,47 @@ bool PlainTextModel::readLines(
     return false;
 }
 
+QTextCodec *PlainTextModel::detectCodec(const char *encodedCharacters, qint64 length)
+{
+    QLocale locale;
+    QTextCodec *codec = QTextCodec::codecForLocale();
+
+    UErrorCode error = U_ZERO_ERROR;
+    if (UCharsetDetector * const detector = ucsdet_open(&error)) {
+        ucsdet_setText(detector, encodedCharacters, length, &error);
+
+
```

Comment:
ETOOMANYEMPTYLINES :)

Reply:
Yes, though these lines of code need separation, there are so many empty lines. I've done it.